x-pack/plugins/security/public/account_management/user_profile/user_profile.tsx

```diff
@@ -210,55 +211,67 @@ function UserSettingsEditor({
       <FormRow
         name="data.userSettings.darkMode"
         label={
-          <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
-            <EuiFlexItem grow={false}>
-              <FormLabel for="data.userSettings.darkMode">
-                <FormattedMessage
-                  id="xpack.security.accountManagement.userProfile.userSettings.theme"
-                  defaultMessage="Mode"
-                />
-              </FormLabel>
-            </EuiFlexItem>
-            <EuiFlexItem grow={false}>{renderHelpText(isThemeOverridden)}</EuiFlexItem>
-          </EuiFlexGroup>
+          <FormLabel for="data.userSettings.darkMode">
+            <FormattedMessage
+              id="xpack.security.accountManagement.userProfile.userSettings.theme"
+              defaultMessage="Mode"
+            />
+          </FormLabel>
```

Comment:
Assuming you apply my previous suggestion to apply a legend to the `EuiKeyPadMenu`, you should be able to remove this `label` prop altogether.